tests/kernel/usb_drive.pm

```diff
@@ -49,8 +54,34 @@ sub run {
     assert_script_run "mount -t btrfs $device $mountpoint";
     assert_script_run "cd $mountpoint; md5sum -c $md5; cd /";
 
-    # cleanup
     assert_script_run("umount $mountpoint");
+
+    if (zypper_search('lklfuse')) {
+        $mountpoint = '/home/unpriv/mount';
+        $file = '/home/unpriv/file2';
+        $md5 = '/home/unpriv/md5';
+        $file_copy = "$mountpoint/file2";
+
+        assert_script_run "useradd -m -G users,disk unpriv -p $testapi::password";
+        install_package 'lklfuse';
+
+        script_start_io('su unpriv');
+        assert_script_run "dd if=/dev/urandom of=$file bs=1M count=16";
+        assert_script_run "md5sum $file > $md5";
+        assert_script_run "mkdir -p $mountpoint";
+        assert_script_run "lklfuse -o type=btrfs $device $mountpoint";
+        assert_script_run "cp $file $file_copy";
+        assert_script_run "fusermount -u $mountpoint";
+        assert_script_run "sync";
+        assert_script_run "lklfuse -o type=btrfs $device $mountpoint";
+        assert_script_run "cd $mountpoint; md5sum -c $md5; cd ";
```

Comment:
This assert will only check the exit code of the last `cd`. You should either split it up into separate asserts or use `&&`.